zio-http/src/main/scala/zio/http/CookieEncoder.scala

```diff
@@ -1,41 +1,52 @@
 package zio.http
 
 import io.netty.handler.codec.http.{cookie => jCookie}
+import zio.Unsafe
 import zio.http.Cookie.SameSite
 
 sealed trait CookieEncoder[A] {
-  final def apply(a: Cookie[A]): String = unsafeEncode(a, false)
-  def unsafeEncode(a: Cookie[A], validate: Boolean): String
+  final def apply(a: Cookie[A]): String =
+    Unsafe.unsafe { implicit u => unsafe.encode(a, validate = false) }
```

Comment:
The unsafety of the encoder is not handled here

Reply:
I think we should require `implicit unsafe: Unsafe` here, then.